frontend/src/components/TitleBar/Tags/ColorEditor.module.css

```diff
@@ -1,3 +1,22 @@
+@media (max-width: 500px) {
+  .ColorEdit {
+  
+    width: 54px !important;
+    font-size: 0px !important;
+    float: right;
+    position: absolute !important;
+    right: 70px !important;
+  }
+  
+  .ColorPicker {
+    position: absolute !important;
+    right: 79px !important;
+    top: 50px !important;
+    z-index: 3;
```

Comment:
Are all these `!important` really necessary? 🤔

Reply:
that's fair. I just committed a fix to remove all of the unnecessary !important rules.